api/SVGElement.json

```diff
@@ -55,40 +55,40 @@
           "spec_url": "https://dom.spec.whatwg.org/#eventdef-abortsignal-abort",
```

Comment:
You didn't add this, but this spec URL is wrong, and furthermore I can't find that this event would ever be fired.

As far as I can tell, `onabort` is on `SVGElement` only because it's in `GlobalEventHandlers`, and nothing would ever fire such an event on any `SVGElement`, the only type of `Element` were it's fired is `HTMLMediaElement`.

So I think we should remove this entry and https://developer.mozilla.org/en-US/docs/Web/API/SVGElement/abort_event.

Reply:
Removing this and the other three events in #11071!